src/App.js

```diff
@@ -43,6 +44,7 @@ const routes = [
     createRoute("/organization/offers", withUserAuth(OffersView, ["organization"], {getOffers: () => offersAPI.getOffersFromOrganization(), buttons: hrOfferButtons})),
     createRoute("/login", <LoginForm />),
     createRoute("/register", <RegistrationRouting />),
+    createRoute('/task/:id', <TaskWrapper />, {}),
```

Comment:
Ogólnie taka ścieżka pokazowa, na razie nie wiem, czy tak to zostawimy, czy nie będzie trzeba jakoś tego ulepszyć (zahasłować, albo coś), dlatego na razie zostawiam tak. Do pokazówki - ścieżka `task/16.`

Reply:
Ogólnie chyba można by walnąć gdzieś uuid jak na moje

Reply:
ogólnie to id to jest id procesu -  one nie mają uuid, a na ten moment do nich przywiązane są taski

Reply:
Znaczy ja bardziej myślałem żeby dać uuid resultTask i stamtąd brać potem id taska

Reply:
Ale jak masz rozwiązywać dopiero taska, to nie będziesz miał resulta